_sort.py

```diff
@@ -0,0 +1,66 @@
+import os
+import yaml
+
+XT = "/etc/sysconfig/network-scripts/ifcfg-"
+SKIP = { "spaldingwcic0.chtc.wisc.edu.yaml", "unl-svc-1.facility.path-cc.io.yaml",
+         "path-ap2001.chtc.wisc.edu.yaml", "glidein-cm3000.chtc.wisc.edu.yaml"}
+MP = {"vxlan123": ["0601", "1200"],
+      "eth0": ["0600", "0601"],
+      "eth1": ["0600", "0601"],
+      "bond0": ["0601"],
+      "ib0": ["0602"],
+      "br0": ["0601"]}
+
+
+for file in os.listdir("./nodes"):
```

Comment:
What if this dir contains non-YAML files? You should check out the `glob` library